omod-1.10/src/main/java/org/openmrs/module/attachments/rest/ObsByConceptListSearchHandler1_10.java

```diff
@@ -46,7 +46,7 @@
 	protected AttachmentsContext context;
 	
 	private final SearchConfig searchConfig = new SearchConfig("obsByConceptList", RestConstants.VERSION_1 + "/obs",
-	        Arrays.asList("1.10.*", "1.11.*", "1.12.*", "2.0.*", "2.1.*", "2.2.*"),
+	        Arrays.asList("1.10.*", "1.11.*", "1.12.*", "2.0.*", "2.1.*", "2.2.*", "2.3.*"),
```

Comment:
@kml27 can you try with the single value `"1.10.0"` this should mean "from 1.10.0 and above".